com.stansassets.build/Editor/Tasks/IBuildTask.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace StansAssets.Build.Editor
+{
+    public interface IBuildTask
+    {
+        void OnPostprocessScene(PlatformType type);
+        int Priority { get; }
+    }
```

Comment:
@alexey-yaremenko please take a look this is not what we dicussed

Reply:
@nazar-korinnyi this is not according to the architecture we discussed.
we have a block-diagram where the whole architecture is. please, follow up those class interfaces